causalpy/pymc_experiments.py

```diff
@@ -203,8 +219,22 @@ def __init__(
         COORDS = {"coeffs": self.labels, "obs_indx": np.arange(self.pre_X.shape[0])}
         self.model.fit(X=self.pre_X, y=self.pre_y, coords=COORDS)
 
-        # score the goodness of fit to the pre-intervention data
-        self.score = self.model.score(X=self.pre_X, y=self.pre_y)
+        if self.validation_time is None:
+            # We just have pre and post data, no validation data. So we can score the pre intervention data
+            self.score = self.model.score(X=self.pre_X, y=self.pre_y)
```

Comment:
Do you think we could replace on validation score by `bayesian tail prob` instead of R2? So, the interpretation here is about how much the real mean during the validation diverge from the posterior mean.

Reply:
Sure. Just looking into it so that I get it right - I just need the high level algorithm because I've not heard that much about it. Google search for "bayesian tail probability" shows _very_ few hits. Is this a widely used approach? Doesn't matter if not, as long as it does what we want :)